tests/unit/EntityMenu.spec.ts

```diff
@@ -81,47 +82,12 @@ describe('Entity Menu - entities', () => {
     wrapper.destroy()
   })
 
-  it('displays entity info properly for a draft IA', async () => {
```

Comment:
Right. It's no longer Filings UI who displays the dashboard with the special EntityInfo component for draft businesses.